src/config/args.hpp

```diff
@@ -145,6 +145,8 @@
 #define LBA_RECONSTRUCTION_BATCH_SIZE             1024
 
 #if defined (__powerpc64__)
+// POWER ABI requires more number of registers to be saved
+// in stack during context switch.
```

Comment:
This doesn't make sense to me. The context switch requires 168 bytes of stack space. I don't see why that would mean the stack size needs to be doubled. What error do you see? Does is still occur with the revised context switching code?